crates/sui-graphql-rpc/src/types/validator_set.rs

```diff
@@ -5,11 +5,19 @@ use super::big_int::BigInt;
 use super::validator::Validator;
 use async_graphql::*;
 
+/// Representation of `0x3::validator_set::ValidatorSet`.
 #[derive(Clone, Debug, PartialEq, Eq, SimpleObject, Default)]
 pub(crate) struct ValidatorSet {
+    /// Total amount of stake for all active validators at the beginning of the epoch.
```

Comment:
it doesnt seem like these comments are appearing in the schema?

Reply:
Ah, that was a bad rebase on my part, I think -- thanks for spotting.